client/src/components/History/Content/ContentItem.vue

```diff
@@ -176,6 +177,7 @@ export default {
             if (this.isCollection) {
                 return {
                     edit: `collection/edit/${id}`,
+                    showDetails: this.item.job_source_id ? `jobs/${this.item.job_source_id}/view` : null,
```

Comment:
I think you need to dispatch here on where you got a job or a implicit collection job ... and in fact we don't have a implicit collection job info page. You need to pick another tool for the selenium test, one that creates a collection not by mapping over a collection.

Reply:
Thank you so much for the help Marius! now It all makes sense! :)